R/sf.R

```diff
@@ -178,11 +178,11 @@ GeomSf <- ggproto("GeomSf", Geom,
 
 default_aesthetics <- function(type) {
   if (type == "point") {
-    GeomPoint$default_aes
+    aes(shape = 19, colour = "black", size = 1.5, fill = NA, alpha = NA, stroke = 0.5)
```

Comment:
Why can we no longer inherit from point?

Reply:
Because with the `expr()` wrapper the `default_aes` no longer played nice with `defaults()` function or `utils::modifyList()` which is used by `sf`. I expect this can/will be reverted with this round of changes.

Reply:
Actually now that I'm looking at this again, geom_sf can't inherit any "themeable" aesthetics because this function (and thus these aesthetics) get evaluated in `draw_geom` step of `layer.r` (rather than earlier in the `get_defaults` step as do all other geoms), so it currently passes an unevaluated expression to `grDevices::col2rgb()` which predictably gives an error . So this needs some refactoring to be "themeable" - either by evaluating aesthetics at the proper step (rather than passing NULL values as geom_sf does now) or by pulling theme into `draw_geom` so that the sf aesthetics can be evaluated in the context of the plots theme. It would seem to me that the former is a preferable solution but I expect that it was written this way for a reason that will become obvious as I dig in. I'm fixing your other concerns now and will work on sf in a future commit.